plugin-trino/src/test/java/org/apache/ranger/authorization/trino/authorizer/RangerSystemAccessControlTest.java

```diff
@@ -132,7 +132,14 @@ public void testTableOperations()
     assertEquals(accessControlManager.filterTables(context(alice), aliceCatalog, aliceTables), aliceTables);
     assertEquals(accessControlManager.filterTables(context(bob), "alice-catalog", aliceTables), ImmutableSet.of());
 
-    accessControlManager.checkCanCreateTable(context(alice), aliceTable,Map.of());
+    accessControlManager.checkCanCreateTable(context(alice), aliceTable, Map.of());
+    assertThrows(AccessDeniedException.class, () -> {
+      accessControlManager.checkCanCreateTable(
+        context(alice),
+        new CatalogSchemaTableName("alice-catalog", "schema","wrong-table"),
+        Map.of()
+      );
+    });
```

Comment:
Since they use guava ImmutableSet, maybe we should use guava ImmutableMap (which also needs to be imported). I don't know if they are picky around this.

```suggestion
    accessControlManager.checkCanCreateTable(context(alice), aliceTable, ImmutableMap.of());
    assertThrows(AccessDeniedException.class, () -> {
      accessControlManager.checkCanCreateTable(
        context(alice),
        new CatalogSchemaTableName("alice-catalog", "schema", "wrong-table"),
        ImmutableMap.of()
      );
    });
```

Reply:
I copied that from exisiting tests. I rather not change this.

Reply:
Ok, can you at least add a space before "wrong-table" :)